lms/templates/discussion/index.html

```diff
@@ -22,6 +22,13 @@
 ## Enable fast preview to fix discussion MathJax rendering bug when page first loads.
 <%include file="_js_body_dependencies.html" args="disable_fast_preview=False"/>
 <%static:js group='discussion'/>
+<script language="javascript">
+   $(function () {
+       $("section.discussion").each(function (index, elem) {
+           DiscussionApp.start(elem, true);
+       });
+   });
+ </script>
```

Comment:
I'm unclear on why this is needed (and what the "the 3rd party tool that should not be named" is), but we should avoid script tags in HTML if at all possible. It's not testable (or lintable) and takes us further from the goal of getting all of Discussions to be loaded with RequireJS.

The better solution is to create a factory and load it with the mako `require_module` function. [See Andy's `DiscussionBoardFactory` here](https://github.com/edx/edx-platform/pull/12772/files#diff-b9a60134e0d9d993486faace7ec1513fR31). Again, not really clear what exactly this does so I can't give more specific direction at the moment, but I'll see what I can do if you fill me in a little more.